pkg/analysis/statussubresource/analyzer.go

```diff
@@ -158,3 +153,7 @@ func hasStatusField(sTyp *ast.StructType, jsonTags extractjsontags.StructFieldTa
 
 	return false
 }
+
+func kubebuilderFormatWithValue(format string) string {
+	return fmt.Sprintf("%s:=%s", markers.KubebuilderRootMarker, format)
+}
```

Comment:
I think this would be better. Note though, that not all markers support the `:=` syntax, so this isn't universal. Anything with Kubebuilder in the name does support `:=` though
```suggestion
func formatKubeBuilderMarkerWithValue(marker, value string) string {
	return fmt.Sprintf("%s:=%s", marker, format)
}
```

Reply:
Fixed!